modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-dpa-multiple-bsl_{context}"]
+= Configuring the DPA with more than one BSL
+
+You configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must create the secrets by using the credentials provided by the cloud provider.
+
+.Procedure
+
+. Configure the DPA with more than one BSL. See the following example.
++
+.Example DPA
+[source, yaml]
+----
+#...
+backupLocations:
+  - name: aws # <1>
+    velero:
+      provider: aws
+      default: true
```

Comment:
later, in the example, this is explained, but would also add note here that if BSL is not selected during backup creation, default BSL is used

Reply:
This IS a very important point, thank you @mateusoliveira43

Reply:
Got it. Will add a note here. Note will say something along the lines of  - If BSL is not set then default is used and at a time only one BSL can be set as default.

Reply:
done.